.github/actions/integration-tests/action.yml

```diff
@@ -0,0 +1,42 @@
+name: Run cellarium-cas integration tests
+description: Runs the integration tests from the cellarium-cas repository
+inputs:
+  revision:
+    description: The revision of the cellarium-cas repository to use
```

Comment:
This _may_ change once we're publishing to pypi

Reply:
Oh really?  What will change?

Reply:
That we might want to pip install from `cellarium-cas` rather than a git hash

Reply:
Would that prevent us from running the integration tests for arbitrary git hashes of the client repo?  I think we want to have the ability to do that

Reply:
Yeah, that's a fantastic point.  I retract my comment :)